im-sdk-core/src/main/java/com/easemob/im/server/api/metadata/user/get/MetadataGetUser.java

```diff
@@ -1,20 +1,22 @@
 package com.easemob.im.server.api.metadata.user.get;
 
 import com.easemob.im.server.api.Context;
+import com.easemob.im.server.model.EMMetadata;
 import reactor.core.publisher.Mono;
 
-public class MetadataGet {
+public class MetadataGetUser {
```

Comment:
Here MetadataGet is good enough, since it could contains several methods:
- getMetadataFromUser
- getMetadataFrom...